### PR TITLE
Push WIP schema to github for test PR

### PR DESCRIPTION
**Pull request by ecuyle**
@Ceciil @rghosh8 
Please review this PR for test purposes.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+USE lairdnd;
+
+/* -------------------------------------------------------*/
+/*                      Create tables                     */
+/* -------------------------------------------------------*/
+
+CREATE TABLE photos (
+	id INT AUTO_INCREMENT PRIMARY KEY,
+	offerings_id INT,
+	url VARCHAR(255),
```
Comment (anlee3) [inline schema.sql:13]: This is a test comment.

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,179 @@
+DROP DATABASE IF EXISTS lairdnd;
+CREATE DATABASE lairdnd;
+
+USE lairdnd;
+
+/* -------------------------------------------------------*/
+/*                      Create tables                     */
+/* -------------------------------------------------------*/
+
+CREATE TABLE photos (
```
Comment (rghosh8) [inline schema.sql:10]: This is a test comment

**Review by anlee3 — CHANGES_REQUESTED** [review]

**Review by rghosh8 — APPROVED** [review]

File: schema.sql (hunk 1 above)
Comment (ecuyle) [reply schema.sql:13]: Hey Angela, I disagree as this is only a test. Will work on the rest of the script later. Please approvel

Comment (anlee3) [reply schema.sql:13]: Ok, looks good!

**Review by anlee3 — APPROVED** [review]

**Review by anlee3 — APPROVED** [review]

**Review by rghosh8 — APPROVED** [review]

**Review by rghosh8 — APPROVED** [review]

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,179 @@
+DROP DATABASE IF EXISTS lairdnd;
```
Comment (anlee3) [inline schema.sql:1]: Please fix this test here.

**Review by anlee3 — CHANGES_REQUESTED** [review]
Please check the comments. Thanks!

File: schema.sql (hunk 3 above)
Comment (ecuyle) [reply schema.sql:1]: As the comment above was a test comment, resolving now.

**Review by anlee3 — APPROVED** [review]
I approve of this.

**Pull request merged**